content/specific-skill-success-criteria/basic-data-analysis/_index.md

```diff
@@ -5,4 +5,334 @@ title: 'Assessment: Basic data analysis'
 topic_needs_review: true
 ---
 
-TODO
\ No newline at end of file
+All students must understand all of the following concepts:
+
+- Statistical concepts:  Mean, Median and Mode
+- Statistical concept:  IQR (Interquartile range)
+- Statistical concepts:  Standard deviation and Variance
+- Statistical concepts:  Correlation and p-value (Probability value) and Null-hypothesis vs. Alternative-hypothesis
+- Statistical concepts:  Standard deviation and Standard error
+- Python concepts:  Thirteen questions and how you would answer them using Python and Pandas
+
+### Statistical concepts: Mean, Median and Mode
+
+#### Mean (Average value):
+```
+import numpy
```

Comment:
Would it make sense to ask the student how to calculate the Mean, median and mode without using code?

Eg: 

```
if you have a list of 5 numbers, eg: 23, 52,12,1,8, how would you calculate the mode? What is the mode? 
```

One difficulty with ds folks is that some of them dont understand what's happening under the hood. They can call functions and use lingo, but dont actually know what's up.